goalgorilla/open_social
# PROD-31065: [IOSA-SUP] Reported content cannot be seen or deleted from the Report Overview Page

## Problem (for internal)
Reported content cannot be seen or deleted from the Report Overview Page by a Site Admin role. Please see the attachment, there is no option to delete and also when clicked on the view comment it doesn't direct to the comment, the user stays at the page. If we try to delete the reported content from the post itself, it gives the following message and do not delete. {"message":"The "comment" parameter was not converted for the path "/ajax_comments/{comment}/delete" (route name: "ajax_comments.delete")"} Reported contents can only be deleted from content overview page.

## Solution (for internal)
Rework the reports page to add a delete option, filters and fix errors on the view action.
The destination parameter was removed from action links because it breaks a redirect that exists on routes.

## Release notes (to customers)
The Reports page (/admin/content/reports) was updated, direct link to remove content and filters by type and status were added.

## Issue tracker
https://getopensocial.atlassian.net/browse/PROD-31065

## Theme issue tracker


## How to test
- [x] Install Open Social
- [x] Report Content
- [x] Go to /admin/content/reports page
- [x] Filters and action links should work without errors

**There was another PR https://github.com/goalgorilla/open_social/pull/4190 that was declined because the update path was not provided, the only difference now is hook update, so we need to check only this.**

## Change Record


## Translations
